### PR TITLE
Don't use BCPInfo to scale down

### PR DESCRIPTION
**Pull request by bratseth**

### CODE REVIEW

**Comment by bratseth** [comment]
I need to look into one more thing before merging.

**Comment by bratseth** [comment]
Good to go now.

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/ClusterModel.java [hunk 1]
```diff
@@ -247,16 +247,21 @@ public Load loadWith(int givenNodes, int givenGroups) {
      */
     public Load idealLoad() {
         var ideal = new Load(cpu.idealLoad(), memory.idealLoad(), disk.idealLoad()).divide(redundancyAdjustment());
+        System.out.println("---------------------------------");
+        System.out.println("Initial ideal: " + ideal);
```
Comment (freva) [inline node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/ClusterModel.java:251]:
```suggestion
```

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/ClusterModel.java [hunk 2]
```diff
@@ -247,16 +247,21 @@ public Load loadWith(int givenNodes, int givenGroups) {
...
         if ( !cluster.bcpGroupInfo().isEmpty() && cluster.bcpGroupInfo().queryRate() > 0) {
             // Since we have little local information, use information about query cost in other groups
             Load bcpGroupIdeal = adjustQueryDependentIdealLoadByBcpGroupInfo(ideal);
 
+            System.out.println("Bcp group ideal: " + bcpGroupIdeal);
```
Comment (freva) [inline node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/ClusterModel.java:256]:
```suggestion
```

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/ClusterModel.java [hunk 3]
```diff
@@ -247,16 +247,21 @@ public Load loadWith(int givenNodes, int givenGroups) {
...
             // Do a weighted sum of the ideal "vote" based on local and bcp group info.
             // This avoids any discontinuities with a near-zero local query rate.
             double localInformationWeight = Math.min(1, averageQueryRate().orElse(0) /
                                                         Math.min(queryRateGivingFullConfidence, cluster.bcpGroupInfo().queryRate()));
             ideal = ideal.multiply(localInformationWeight).add(bcpGroupIdeal.multiply(1 - localInformationWeight));
+            System.out.println("local information weight: " + localInformationWeight);
         }
+        System.out.println("Adjusted ideal: " + ideal);
```
Comment (freva) [inline node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/ClusterModel.java:264]:
```suggestion
        }
```

**Review by freva — DISMISSED** [review]

**Review by freva — APPROVED** [review]

**Pull request merged**